src/Runtime/localtest/src/Services/TestData/TestDataService.cs

```diff
@@ -56,14 +64,19 @@ void PopulateDefaults(TestDataModel model)
                 var tenorUsers = await _tenorDataRepository.GetAppTestDataModel();
                 if (tenorUsers is not null && !tenorUsers.IsEmpty())
                 {
-                    // Use tenor users if they exist
```

Comment:
I wish I remembered what I was thinking here. It does not make sense to me why tenor users would be relevant here.

Reply:
Rest assured, I remember less! :shrug: